src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1178,7 +1178,7 @@ public class MsmqDequeueStrategy
     }
 }
 
-namespace NServiceBus.Unicast.Transport
+namespace NServiceBus.Unicast.Transports
```

Comment:
I checked the file on the branch manually and it also contains the namespace renamings, so it doesn't seem to be a caching issue. Let's talk again tomorrow to get that merged.